blenderproc/python/loader/HavenMaterialLoader.py

```diff
@@ -128,6 +143,8 @@ def load_haven_mat(folder_path: str = "resources/haven", used_assets: list = [],
             new_mat = MaterialLoaderUtility.find_cc_material_by_name(texture_name, add_cp)
         else:
             new_mat = MaterialLoaderUtility.create_new_cc_material(texture_name, add_cp)
+        # append newly created material
+        materials.append(new_mat)
```

Comment:
I think the materials here are from type bpy.types.Material, so you need to wrap them first in our material class

Reply:
Yes, you are right. I added the missing return types to the `MaterialLoaderUtility` fcts. to avoid such errors in the future.

Reply:
Nice, but you now probably need to adjust the other usages of these methods

Reply:
What do you mean? I only added type hints, so that such errors at least raise a warning in the IDE.

Reply:
Ah yes, sorry I missed that. Then its fine :+1: